_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: single
+title: "Big Changes for pyOpenSci - an update from the new executive director, leah wasser, what's next?"
+excerpt: "pyOpenSci has recently transitioned to a fiscal sponsor and has a new executive director - Leah Wasser. Learn more about our goals over the next few years and how to get involved with the community."
+author: "Leah Wasser"
+permalink: /blog/new-executive-director-leah-wasser
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+---
+
+
+## pyOpenSci is now an independent community organization
+
+I am thrilled to announce that pyOpenSci is now a fiscally sponsored project of 
+Community Initiatives! This means that pyOpenSci now serves the community of 
+scientific python users and open source software maintainers. pyOpenSci is not 
+affiliated with any specific organization. I (Leah) will now be devoting ALL of my 
+(work :)) time to the pyOpenSci project full time as the official Executive Director. 
```

Comment:
I love parenthetical injections but maybe use an emoji since ":))" is hard to parse (some people use that for "laughing")

Reply:
huh i will see if i can add an emoji to markdown! never tried that

Reply:
:grin:

Reply:
vs code no likey but gh might.